packages/.idea/workspace.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
https://github.com/DeFiCh/jellyfish/pull/581/files#r689420766

Reply:
My bad, did not see it pass through. Let me remove it